Resources/Prototypes/Entities/Objects/Specific/chemistry.yml

```diff
@@ -38,7 +37,7 @@
   - type: Pourable
     transferAmount: 5.0
   - type: Spillable
-  - type: GlassBeaker
```

Comment:
![grafik](https://user-images.githubusercontent.com/16618384/101892537-cd990e00-3ba3-11eb-8346-3f0b4d0b0d89.png)

Reply:
I have zero idea why the beaker decided not to be a beaker anymore